### PR TITLE
Add validations for docker compose env files

### PR DESCRIPTION
**Pull request by punchagan**

### CODE REVIEW

File: scripts/validate-env.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    then
+        echo "$1 is valid"
+    else
+        echo "$1 is NOT valid"
+        ERRORS="yes"
+    fi
+}
+
+is_graphql_url() {
+    regex='https?://.*/v1/graphql'
```
Comment (gs0510) [inline scripts/validate-env.sh:42]: This also 😿

File: scripts/validate-env.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    if [ -d $1 ]; then
+        echo "$1 exists"
+    else
+        echo "${1} is NOT a valid directory"
+        ERRORS="yes"
+    fi
+}
+
+url_with_schema() {
+    regex='https?://.*'
```
Comment (gs0510) [inline scripts/validate-env.sh:31]: this should be `http` 😬

**Review by gs0510 — CHANGES_REQUESTED** [review]
looks great thanks @punchagan!!

File: scripts/validate-env.sh (hunk 2 above)
Comment (punchagan) [reply scripts/validate-env.sh:31]: `s?` would mean that `s` either appears, or not.

**Review by gs0510 — APPROVED** [review]

**Pull request merged**